Week3/week3_homework/exercise4.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const groceries = ['bananas', 'milk'];
+
+function addToShoppingCart(item) {
+  groceries.push(item);
+
+  if (groceries.length > 3) {
```

Comment:
Ah. I think there is a slight mistake here. Requirement was to ensure that there are no more than 2 items in the cart. But we can have 3 now with current logic. 🙂